src/components/vm/nics/vmNicsCard.jsx

```diff
@@ -527,7 +528,7 @@ export class VmNetworkTab extends React.Component {
         let networkId = 1;
         detailMap = detailMap.filter(d => !d.hidden);
 
-        const columnTitles = detailMap.map(target => target.name);
+        const columnTitles = detailMap.map(target => ({ title: target.name, props: { "aria-label": target.aria } }));
```

Comment:
Aria can be null here right? So we should probably not pass an aria-label which is null.